include/flatbuffers/flatbuffers.h

```diff
@@ -1523,9 +1529,9 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
   const uint8_t *buf_;
   const uint8_t *end_;
   size_t depth_;
```

Comment:
`depth_` and `max_tables_` also will need to be changed, otherwise you'll get warnings on other platforms when these values are compared. Besides, on Arduino, `num_tables_` could wrap around before it reaches `max_tables_`.